code/API_definitions/qos-profiles.yaml

```diff
@@ -231,8 +231,7 @@ components:
         maxDuration:
           description: |
             The maximum time period that this profile can be deployed.
-            NOTE: currently the duration within `sessionInfo` is limited to 86400 seconds (1 day).
-            The value of `maxDuration` shouldn't therefore exceed this time period. The limitation might be removed in later versions.
+            Overall session duration must not exceed this value. This includes the initial requested duration plus any extensions.
           allOf:
             - $ref: "#/components/schemas/Duration"
```

Comment:
Within quality-on-demand we are using seconds for duration, here we have the duration object. For this PR that's ok for me (e.g. maxDuration can be expressed in seconds), a potential consolidation with a common duration format is a separate discussion (see https://github.com/camaraproject/Commonalities/issues/137).

Reply:
We can adapt the APIs to the outcome of the discussion. IMO the qos-profile API is more oriented to inform clients about the characteristics of the QoS Profile, with a human friendly response, so we could decide to maintain the current format, but I don't have a strong opinion. I doubt it will be used to discover profiles dynamically. My assumption is that customers will likely know which profile to use before using the API, as that would be part of the service offering, but that is a different discussion.

Reply:
Created https://github.com/camaraproject/QualityOnDemand/issues/304 for this point as discussed within QoD call on June 14th.